### PR TITLE
Virt resize error if image is already expand and the vm is running

### PR DESCRIPTION
**Pull request by jamesregis**

### CODE REVIEW

File: molecule_libvirt/playbooks/destroy.yml [hunk 1]
```diff
@@ -135,6 +135,15 @@
         label: "{{ item.name }}"
       delegate_to: "{{ item.libvirt_host if item.libvirt_host is defined else 'localhost' }}"
 
+    - name: Remove hidden expanded info file
+      ansible.builtin.file:
+        path: "{{ libvirt_default_image_path }}/.{{ item.name }}"
```
Comment (ssbarnea) [inline molecule_libvirt/playbooks/destroy.yml:140]: That is missing the `-expanded` bit, will not remove what you want

File: molecule_libvirt/playbooks/templates/vm/vm.xml.j2 [hunk 2]
```diff
@@ -21,7 +21,7 @@ or other application using the libvirt API.
     <vmport state='off'/>
   </features>
   <cpu mode='custom' match='exact' check='none'>
-    <model fallback='allow'>{{ item.cpu | default('qemu64')}}</model>
+    <model fallback='allow'>{{ item.cpu_model | default('qemu64')}}</model>
```
Comment (ssbarnea) [inline molecule_libvirt/playbooks/templates/vm/vm.xml.j2:24]: That seem unrelated, if is a fix it should be done as separated PR so we can easily track them.

Comment (jamesregis) [reply molecule_libvirt/playbooks/templates/vm/vm.xml.j2:24]: My bad. I have commited this change in the wrong branch.

**Review by ssbarnea — APPROVED** [review]
